### ISSUE TITLE
Display aspect ratio in resolution select dialog

### ISSUE THREAD

**Issue by Osmodium** [ihhub/fheroes2#6586]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

I never know which resolution uses a certain aspect ratio, also since I change the screen I play on from time to time, and if the resolutions and scales are added dynamically then you might want to know if a resolution would look good on a large variety of monitors.

### Describe the possible solution

To show the aspect ratio in the screen resolution dialog would help the player choose a fitting resolution.
It could look something like this;
![image](https://user-images.githubusercontent.com/234248/217097408-73b2ca09-89f4-4648-abd9-f994d87e8757.png)

**Comment by oleg-derevenetz** [comment]
Hi @Osmodium if you already have an implementation, you can offer a pull request. I believe this improvement might be really handy (the specific design can be discussed in the process).

**Comment by Osmodium** [comment]
@oleg-derevenetz I have an implementation, and yea it might not be the prettiest as of now, but I will go ahead and make a pr :)

**Comment by Osmodium** [comment]
PR opened here https://github.com/ihhub/fheroes2/pull/6587

**Comment by oleg-derevenetz** [comment]
Hi @Branikolog @zenseii what do you think about this design?

**Comment by Branikolog** [comment]
Hi, @oleg-derevenetz !
For me personally, the gap between these two values is too large. I prefer them be placed closer to each other. Or adding a line in this gap connecting them. We can also align columns with respect to " x " and " : " being in center.

Or even don't change anything here and simply add this aspect ratio value into info window when we right click on some particular resolution.

**Comment by Osmodium** [comment]
I would agree that the gap might be too big, however for me to quickly see which resolutions are which ratios is harder if I'd have to right-click each individually.
I can see that the format of the lines differ depending on if there is scaling involved, but I'd be happy to make a pass of this where they are aligned closer with a format that would look something like this:
Where scale equals 1: `1920 x 1080 (16 : 9)`
Where scale larger than 1: `1920 x 1080 (16 : 9) (x2)`

**Comment by Osmodium** [comment]
I have tried a few different iterations of different layouts.
A) Which might be too cramped
![image](https://user-images.githubusercontent.com/234248/217279114-1654b318-82f9-46cf-a1d2-33926d845aff.png)

B + C) Without and with scale
![image](https://user-images.githubusercontent.com/234248/217279241-f780475b-ec95-4f78-924d-efde7c855f9c.png)
![image](https://user-images.githubusercontent.com/234248/217279730-d983e632-157b-48c7-a00c-414a6bfdea6f.png)

D + E) Without and with scale
![image](https://user-images.githubusercontent.com/234248/217279760-5f4874e9-09fd-4dd8-a754-2e2d26e7503e.png)
![image](https://user-images.githubusercontent.com/234248/217279779-e35b42cb-4190-40b0-a63c-43ae30c42823.png)

**Comment by Laserlicht** [comment]
Visually the last variant looks best I think.

**Comment by ihhub** [comment]
I recommend to use the current implementation in must branch when we show scaling only for x2 and higher and add proportions on the right side. We need to remember that visual information must be minimal and having x1 everywhere can easily lead to missing x2 sign.

**Comment by Osmodium** [comment]
@ihhub yes, the `x1` was just for reference since I don't have any `x2` in my list.
